### PR TITLE
Added .ruby-version file to project

### PR DESCRIPTION
**Pull request by dkniffin**
## Why?

- #74

## What Changed?

- Updated code to generate a .ruby-version file

### CODE REVIEW

**Review by dacur — APPROVED** [review]
👍

File: spec/support/shared_examples/basic_setup_steps.rb [hunk 1]
```diff
@@ -31,6 +31,10 @@
     expect(travis_file).to match(/^  - bundle exec rspec$/)
   end
 
+  it "creates a .ruby-version file" do
+    expect(File).to exist("#{project_path}/.ruby-version")
+  end
+
```
Comment (leeallen337) [inline spec/support/shared_examples/basic_setup_steps.rb:37]: Just curious and I have no idea how possible this is but would it be worth it to have a test to make sure the `.ruby-version` file actually says the right version? If that makes sense

**Review by leeallen337 — APPROVED** [review]
One quick question but 👍

File: spec/support/shared_examples/basic_setup_steps.rb (hunk 1 above)
Comment (dkniffin) [reply spec/support/shared_examples/basic_setup_steps.rb:37]: I thought about that, but I can't think of an easy way to do it.

**Pull request merged**